.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Release
+        uses: cycjimmy/semantic-release-action@v2
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          extra_plugins: |
+            @semantic-release/git@7.0.18
```

Comment:
FYI these versions are pretty old I think, just saw a recent PR from @maetolay updating these in the .github repo

Reply:
We actually copied it from another repo were @bludot had fixed it. Wondering if we could somewhat pin this version across all repos.